services/hooks/src/api.js

```diff
@@ -664,32 +662,38 @@ builder.declare({
   title: 'Get information about recent hook fires',
   stability: 'stable',
   category: 'Hook Status',
+  query: paginateResults.query,
   description: [
     'This endpoint will return information about the the last few times this hook has been',
     'fired, including whether the hook was fired successfully or not',
+    '',
+    'By default this endpoint will return up to 1000 last fires in one request.',
```

Comment:
Let's include that these are the most recent.

Before this change, the call would return _all_ last fires.  Now, the same call (without pagination query) will return _some_ last fires.  But since those are the most recent, and TC periodically flushes the less recent last-fires, I think that's a legitimate change.

Reply:
Thanks! 
Regarding the flushing of last fires I'm not so sure this is happening, as there are few hooks with thousands of those last fires. 
upd: yes, the `expire_last_fires` only deletes those older than 1 year, so apparently it can still get a lot of tasks:

<img width="511" alt="image" src="https://user-images.githubusercontent.com/83861/205274287-b76bbf82-5a8f-4804-9e4f-d9ba8a44d701.png">

Reply:
Ah, right -- IIRC we thought about deleting if there are more than N for a hook, but that's a more complicated query.  That might be worth considering anyway?  In another PR of course :)

Reply:
ah, that's actually another possible solution for this problem.. and an easier one. :)
I'll create a ticket to add such kind of expiration by number as well

Reply:
https://github.com/taskcluster/taskcluster/issues/5839  added here